Java/WhatIsNumberGame.java

```diff
@@ -0,0 +1,94 @@
+package br.com.jacksonroberio;
```

Comment:
Remove package name, this file is independent of package :)

Reply:
ok, done